packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ScmMethodProviderCollection.cs

```diff
@@ -259,6 +259,10 @@ private ValueExpression GetResultConversion(ScopedApi<ClientResult> result, CSha
             {
                 return result.GetRawResponse().Content().ToObjectFromJson(responseBodyType);
             }
+            if (responseBodyType.IsEnum)
+            {
+                return responseBodyType.ToEnum(result.GetRawResponse().Content().ToObjectFromJson(responseBodyType.UnderlyingEnumType));
```

Comment:
Minor. `CSharpType.ToEnum` seems not consistent with other `ValueExpression` API in styles. Maybe we could have something like `ObjectInitializationExpression.CreateEnum`?

Reply:
Spoke offline and sticking with this style for now.